Branch-SDK/src/main/java/io/branch/referral/StoreReferrer.java

```diff
@@ -136,26 +112,26 @@ private static void processReferrerInfo(Context context, String rawReferrerStrin
                 if (referrerMap.containsKey(Defines.Jsonkey.LinkClickID.getKey())) {
```

Comment:
@gdeluna-branch this applies to all App store referrers right? and not just google?

Reply:
That's correct, each store if called will go through the same flow as before